### ISSUE TITLE
Tinker Rearm also rearms the ulti hence the cooldown doesn't work

### ISSUE THREAD

**Issue by R0b0kill** [ValveSoftware/Dota2-Gameplay#15944]
### Ability name

Rearm

### Description

Rearm on Tinker also resets cooldown on Rearm. In other words the normal 8 seconsds cooldown does nothing. Had to watch a replay to understnad how Tinker was so OP

### Example Match ID (and possibly Timestamp)

7637339123 14/3/2024 23:24

**Comment by Aluerie** [comment]
for cancels

**Comment by Hex2D2** [comment]
Yes this is intended. The cooldown is meant for when the ability is interrupted.

**Comment by R0b0kill** [comment]
? but the comments on rearm also state it does rockets and laser not ulti. Besides back in the day I can recall it didn't do rearm. Its not the same as refresher right?
